src/scripts/expletives.js

```diff
@@ -18,7 +18,6 @@ const expletives = {
 	wanker: "phlonominium",
 	bich: "plinimal",
 	cock: "phloopdie",
```

Comment:
Should not have removed this one:
```suggestion
	cock: "phloopdie",
	slut:  "phliminustrim",
```